text/0000-launcher-sbom.md

```diff
@@ -0,0 +1,151 @@
+
+launcher: an executable that is the `ENTRYPOINT` of the exported OCI image. It is used to start processes at runtime.
+
+# Motivation
+
+[motivation]: #motivation
+
+- Why should we do this? A more complete SBOM for CNB-built images.
+- What use cases does it support? As an end user, I want SBOM information for every component of my software supply
+  chain.
+
+# What it is
+
+[what-it-is]: #what-it-is
+
+The SBOM for a CNB-built image could be broken down as follows:
+
+* **Base image dependencies**
+    * The platform is responsible for providing SBOM information (see the closure
+      of https://github.com/buildpacks/rfcs/pull/211 in favor of https://github.com/buildpacks/rfcs/pull/195). Out of
+      scope for this RFC.
+* **Buildpack-provided dependencies**
+    * SBOM files output by buildpacks are copied to `<layers>/sbom/build/<buildpack-id>`
+      and `<layers>/sbom/run/<buildpack-id>`. `<layers>/sbom/run` is a layer in the final image.
+      See [RFC 0095](https://github.com/buildpacks/rfcs/blob/main/text/0095-sbom.md).
+* **CNB lifecycle**
+    * Currently there is no SBOM information associated to the image. A CycloneDX SBOM is included in lifecycle
+      [releases](https://github.com/buildpacks/lifecycle/releases), but it is left up to individual end users to locate
+      this information. The subject of this RFC.
+
+## Proposed Changes:
+
+* The lifecycle will ship with SBOM files in CycloneDX, SPDX, and Syft formats.
+    * [Lifecycle images](https://hub.docker.com/r/buildpacksio/lifecycle) will include the following SBOM files:
+        * `/cnb/lifecycle.sbom.cdx.json`
+        * `/cnb/lifecycle.sbom.spdx.json`
+        * `/cnb/lifecycle.sbom.syft.json`
+        * `/cnb/launcher.sbom.cdx.json`
+        * `/cnb/launcher.sbom.spdx.json`
+        * `/cnb/launcher.sbom.syft.json`
+    * Analogously, [lifecycle tarballs](https://github.com/buildpacks/lifecycle/releases) will include these files
+      rooted at `./lifecycle` - e.g., `/cnb/lifecycle.sbom.cdx.json` above will be `./lifecycle/lifecycle.sbom.cdx.json`
+      in the tarball.
+
+* Builder authors should include the new SBOM files in builder images.
+    * There shouldn't be any changes required to `pack builder create` as `pack` takes a lifecycle tarball as an input,
+      and the files will be there in the tarball.
+
+* Before `export`, the lifecycle will copy the above files to:
+    * `/cnb/lifecycle.sbom.cdx.json`  -> `<layers>/sbom/build/cnb_lifecycle/sbom.cdx.json`
+    * `/cnb/lifecycle.sbom.spdx.json` -> `<layers>/sbom/build/cnb_lifecycle/sbom.spdx.json`
+    * `/cnb/lifecycle.sbom.syft.json` -> `<layers>/sbom/build/cnb_lifecycle/sbom.syft.json`
+    * `/cnb/launcher.sbom.cdx.json`   -> `<layers>/sbom/run/cnb_lifecycle/sbom.cdx.json`
+    * `/cnb/launcher.sbom.spdx.json`  -> `<layers>/sbom/run/cnb_lifecycle/sbom.spdx.json`
+    * `/cnb/launcher.sbom.syft.json`  -> `<layers>/sbom/run/cnb_lifecycle/sbom.syft.json`
```

Comment:
```suggestion
    * `/cnb/launcher.sbom.cdx.json`   -> `<layers>/sbom/run/cnb_launcher/sbom.cdx.json`
    * `/cnb/launcher.sbom.spdx.json`  -> `<layers>/sbom/run/cnb_launcher/sbom.spdx.json`
    * `/cnb/launcher.sbom.syft.json`  -> `<layers>/sbom/run/cnb_launcher/sbom.syft.json`
```
This might be clearer.